### ISSUE TITLE
Cannot read property 'version' of undefined

### ISSUE THREAD

**Issue by chsrlnd** [Microsoft/react-native-windows#1106]
Hello,

I have done:
```
react-native init test
cd test 
npm install --save-dev rnpm-plugin-windows
react-native windows
```
and I get:

```Scanning 580 folders for symlinks in C:\Users\username\Documents\test\node_modules (23ms)
Reading application name from package.json...
Reading react-native version from node_modules...
Checking for react-native-windows version matching 0.44.*...
C:\Users\username\Documents\test\node_modules\rnpm-plugin-windows\src\common.js:56
        resolve(release.version)
                       ^
TypeError: Cannot read property 'version' of undefined
    at npm.packages.range (C:\Users\username\Documents\test\node_modules\rnpm-plugin-windows\src\common.js:56:24)
    at releases (C:\Users\username\Documents\test\node_modules\npm-registry\endpoints\packages.js:182:5)
    at each (C:\Users\username\Documents\test\node_modules\mana\index.js:314:7)
    at Array.forEach (native)
    at child.all (C:\Users\username\Documents\test\node_modules\mana\index.js:313:11)
    at child.<anonymous> (C:\Users\username\Documents\test\node_modules\mana\index.js:620:13)
    at finished (C:\Users\username\Documents\test\node_modules\assign\index.js:331:17)
    at next (C:\Users\username\Documents\test\node_modules\assign\index.js:378:7)
    at iterators (C:\Users\username\Documents\test\node_modules\assign\index.js:386:7)
    at emits (C:\Users\username\Documents\test\node_modules\assign\index.js:243:14)
```

I noticed the following works, taken from [1062](https://github.com/Microsoft/react-native-windows/issues/1062):
```
react-native init v41 --version 0.41.2
cd v41
npm i --save-dev rnpm-plugin-windows
react-native windows
```

Is it possible to use the latest version of react-native?

Thank you!

**Comment by khowling** [comment]
I worked around this by changing package.json version to "react-native": "^0.43", then npm install

**Comment by matthargett** [comment]
In general, I would suggest using version 42 right now as issues get ironed out with 43 and 44 upstream.

**Comment by rozele** [comment]
Release candidates for 0.43, 0.44 and 0.45 are now available.

**Comment by sayar** [comment]
@chsrlnd has your issue been resolved?

**Comment by vpopescu** [comment]
I get the same thing, a few months later. What was the resolution? Sorry, I'm new to this, and I am not able to read between the lines.

```
C:\source\sample>react-native init test
This will walk you through creating a new React Native project in C:\source\sample\test
'yarn' is not recognized as an internal or external command,
operable program or batch file.
Installing react-native...
Consider installing yarn to make this faster: https://yarnpkg.com
npm WARN deprecated connect@2.30.2: connect 2.x series is deprecated
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN react-native@0.49.1 requires a peer of react@16.0.0-beta.5 but none was installed.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ react-native@0.49.1
added 679 packages in 42.406s
Setting up new React Native app in C:\source\sample\test
Installing React...
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ react@16.0.0-beta.5
added 116 packages in 12.274s
Installing Jest...
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ react-test-renderer@16.0.0-beta.5
+ babel-preset-react-native@4.0.0
+ babel-jest@21.2.0
+ jest@21.2.1
added 312 packages and updated 2 packages in 19.96s
To run your app on iOS:
   cd C:\source\sample\test
   react-native run-ios
   - or -
   Open ios\test.xcodeproj in Xcode
   Hit the Run button
To run your app on Android:
   cd C:\source\sample\test
   Have an Android emulator running (quickest way to get started), or a device connected
   react-native run-android

C:\source\sample>cd test

C:\source\sample\test>npm install --save-dev rnpm-plugin-windows
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ rnpm-plugin-windows@0.2.7
added 148 packages in 16.552s

C:\source\sample\test>react-native windows
Scanning folders for symlinks in C:\source\sample\test\node_modules (31ms)
Reading application name from package.json...
Reading react-native version from node_modules...
Checking for react-native-windows version matching 0.49.*...
C:\source\sample\test\node_modules\rnpm-plugin-windows\src\common.js:59
        resolve(release.version)
                       ^

TypeError: Cannot read property 'version' of undefined
    at npm.packages.range (C:\source\sample\test\node_modules\rnpm-plugin-windows\src\common.js:59:24)
    at releases (C:\source\sample\test\node_modules\npm-registry\endpoints\packages.js:182:5)
    at each (C:\source\sample\test\node_modules\mana\index.js:314:7)
    at Array.forEach (<anonymous>)
    at child.all (C:\source\sample\test\node_modules\mana\index.js:313:11)
    at child.<anonymous> (C:\source\sample\test\node_modules\mana\index.js:620:13)
    at finished (C:\source\sample\test\node_modules\assign\index.js:331:17)
    at next (C:\source\sample\test\node_modules\assign\index.js:378:7)
    at iterators (C:\source\sample\test\node_modules\assign\index.js:386:7)
    at emits (C:\source\sample\test\node_modules\assign\index.js:243:14)


```

**Comment by rozele** [comment]
@vpopescu we haven't published react-native-windows@0.49.* yet, coming later this evening. I'm going to re-open this bug though, because that error message is pretty useless.
